starlark/src/values/mod.rs

```diff
@@ -95,18 +95,35 @@ use std::fmt;
 use std::marker;
 use std::rc::Rc;
 
+#[derive(Clone)]
+enum ValueInner {
+    None(ValueHolder<NoneType>),
+    Bool(ValueHolder<bool>),
+    Int(ValueHolder<i64>),
+    Other(Rc<dyn ValueHolderDyn>),
```

Comment:
Should there be also string here?

Reply:
Not string, because:
* string are expensive to copy (and `Value` copying should be cheap)
* size of `Value` struct would increase from 16 bytes to 32 bytes

Maybe `Rc<String>` but this will be a bit more complicated change than this.